routes/do_vote.js

```diff
@@ -33,28 +36,63 @@ exports.post = function (req, res, next) {
   voteText = voteText.replace(triggerWord + ' ','').toLowerCase();
   console.log('Incoming vote for: ' + voteText);
 
-  var count = 0;
-  _.each(data.votes, function(vote) {
-    if (voteText === vote.voteName) {
-      console.log('vote name matched!');
-      vote.voteCount = vote.voteCount + 1;
-      voteMatch = true;
+  dbActions.getActivePollId(getActivePoll);
+  function getActivePoll(pollId) {
+    console.log('Current Active Pollid: ' + pollId);
+    dbActions.getPoll(pollId, setData);
+    activePoll = pollId;
+  }
+
+  function setData(poll_string) {
+    var pollTitleOnly = false;
```

Comment:
Yes, this was just so I could work with start_vote as is and should get removed once that is writing a poll obj.